.github/workflows/ci-main.yml

```diff
@@ -7,6 +7,13 @@ on:
     branches: [ master ]
   workflow_dispatch:
 
+permissions:
+  contents: write
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  VCPKG_FEATURE_FLAGS: dependencygraph
```

Comment:
I would rather put the env and permissions just on the job that needs it. Also I'm pretty sure the GITHUB_TOKEN part doesn't do anything since that is the default.

Reply:
It's how the documentation said to do it.